### PR TITLE
[ID] Update `History of osu!/2014`

### PR DESCRIPTION
**Pull request by rzkalqdr**
Updated Indonesian translation for the `2014` section for the `History of osu!` article.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/2014/id.md [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
+3. [osu! wiki article - "Performance points"](/wiki/Performance_points)
+4. [osu! newspost [Tom94] (1 Maret 2014) - "Performance ranking for all gamemodes"](https://osu.ppy.sh/home/news/2014-03-01-performance-ranking-for-all-gamemodes)
+5. [osu! forum comment (Resolved Issues) [Pawsu] - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
+6. [osu! wiki article - "Skinning history"](/wiki/Skinning/History)
+7. [osu! newspost [LuigiHann] (3 April 2014) - "osu! Monthly Fanart Contest #1 - Ends this week!"](https://osu.ppy.sh/home/news/2014-04-03-osu-monthly-fanart-contest-1-ends-this-week)
+8. [osu! newspost [LuigiHann] (21 Mei 2014) - "First Official osu! Fanart Contest Results!"](https://osu.ppy.sh/home/news/2014-05-21-first-official-osu-fanart-contest-results)
+9. [osu! newspost [peppy] (18 Juni 2014) - "Download restrictions removed"](https://osu.ppy.sh/home/news/2014-06-18-download-restrictions-removed)
+10. [osu! newspost [Ephemeral] (21 Agustus 2014) - "Restructuring of the BAT"](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat)
```
[the remark this answers is not in the document]
Comment (rzkalqdr) [reply wiki/History_of_osu!/2014/id.md]: did this, except for `newspost` i only changed to `*newspost*` just like in previous paragraphs; also consulted w other Indonesians and came to this conclusion

File: wiki/History_of_osu!/2014/id.md [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
+Terakhir, mengarah pada [peppy](https://osu.ppy.sh/users/2) membuat keputusan yang sulit untuk merilis *osu!text.exe* (sekarang *osu!cuttingedge*) dapat digunakan **siapa saja**. Hal ini memungkinkan pemain yang tidak memiliki [osu!supporter](https://osu.ppy.sh/home/support) untuk melakukan *debug* pada osu! di mana osu! akan diperbarui ke klien mana pun. Itu pun, jika mereka ingin menggunakan fitur eksperimental yang kurang stabil. Setiap fungsi yang ada diberikan di osu!cuttingedge **kecuali** fitur multiplayer (membutuhkan osu!supporter).
 
-## Desember
+## Referensi
 
-"Merry Christmas" seperti yang peppy tulis bersamaan dengan pembaruan klien yang saat itu menambahkan salju (berbentuk ikon mode permainan saat ini) berjatuhan di layar utama!
+1. [osu! *newspost* [peppy] (20 Januari 2014) - "Performance Ranking Maintenance"](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)
+2. [osu! *newspost* [Tom94] (26 Januari 2014) - "New Performance Ranking"](https://osu.ppy.sh/home/news/2014-01-26-new-performance-ranking)
+3. [artikel wiki osu! - "Performance points"](/wiki/Performance_points)
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:37]:
```suggestion
3. [Artikel wiki osu! - "Performance points"](/wiki/Performance_points)
```

File: wiki/History_of_osu!/2014/id.md [hunk 3]
```diff
@@ -1,29 +1,44 @@
...
+4. [osu! *newspost* [Tom94] (1 Maret 2014) - "Performance ranking for all gamemodes"](https://osu.ppy.sh/home/news/2014-03-01-performance-ranking-for-all-gamemodes)
+5. [komentar forum osu! (Resolved Issues) [Pawsu] - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
+6. [artikel wiki osu! - "Skinning history"](/wiki/Skinning/History)
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:40]:
```suggestion
6. [Artikel wiki osu! - "Skinning history"](/wiki/Skinning/History)
```

File: wiki/History_of_osu!/2014/id.md [hunk 4]
```diff
@@ -1,29 +1,44 @@
...
 
-## Desember
+## Referensi
 
-"Merry Christmas" seperti yang peppy tulis bersamaan dengan pembaruan klien yang saat itu menambahkan salju (berbentuk ikon mode permainan saat ini) berjatuhan di layar utama!
+1. [osu! *newspost* [peppy] (20 Januari 2014) - "Performance Ranking Maintenance"](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)
+2. [osu! *newspost* [Tom94] (26 Januari 2014) - "New Performance Ranking"](https://osu.ppy.sh/home/news/2014-01-26-new-performance-ranking)
+3. [artikel wiki osu! - "Performance points"](/wiki/Performance_points)
+4. [osu! *newspost* [Tom94] (1 Maret 2014) - "Performance ranking for all gamemodes"](https://osu.ppy.sh/home/news/2014-03-01-performance-ranking-for-all-gamemodes)
+5. [komentar forum osu! (Resolved Issues) [Pawsu] - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:39]:
```suggestion
5. [Komentar forum osu! (Resolved Issues) [Pawsu] - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
```

File: wiki/History_of_osu!/2014/id.md [hunk 5]
```diff
@@ -1,29 +1,44 @@
...
 
 ## Oktober
 
-Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada bpm beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
+Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada BPM beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:29]:
```suggestion
Berbagai fitur baru ditambahkan ke osu! pada bulan ini, mulai dari layar masuk (*sign-in screen*) baru, sistem *scrolling* [osu!mania](/wiki/Game_mode/osu!mania) baru yang berdasar pada BPM, animasi intro dan outro baru, hingga lagu tema baru yang akan terdengar setiap kali osu! dinyalakan.
```

Comment (Nivalyx) [reply wiki/History_of_osu!/2014/id.md:29]: (i checked the original english article and it appears that the actual paragraph here has been drastically simplified in which a lot of things have been removed)

File: wiki/History_of_osu!/2014/id.md [hunk 6]
```diff
@@ -1,29 +1,44 @@
...
 
-Terakhir, mengarah pada peppy membuat keputusan yang sulit untuk merilis *osu!text.exe* (sekarang *osu!cuttingedge*) dapat digunakan **siapa saja**. Hal ini memungkinkan pemain yang tidak memiliki osu!supporter untuk melakukan *debug* pada osu! di mana osu! akan diperbarui ke klien mana pun. Itu pun, jika mereka ingin menggunakan fitur eksperimental yang kurang stabil. Setiap fungsi yang ada diberikan di osu!cuttingedge **kecuali** fitur multiplayer (membutuhkan osu!supporter).
+Terakhir, mengarah pada [peppy](https://osu.ppy.sh/users/2) membuat keputusan yang sulit untuk merilis *osu!text.exe* (sekarang *osu!cuttingedge*) dapat digunakan **siapa saja**. Hal ini memungkinkan pemain yang tidak memiliki [osu!supporter](https://osu.ppy.sh/home/support) untuk melakukan *debug* pada osu! di mana osu! akan diperbarui ke klien mana pun. Itu pun, jika mereka ingin menggunakan fitur eksperimental yang kurang stabil. Setiap fungsi yang ada diberikan di osu!cuttingedge **kecuali** fitur multiplayer (membutuhkan osu!supporter).
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:31]:
```suggestion
Di samping itu, dalam rangka meningkatkan laju pengembangan dan penumpasan bug (*debugging*) yang ada, [peppy](https://osu.ppy.sh/users/2) juga merilis klien osu! versi "osu!test" (yang kini dikenal dengan nama "Cutting Edge") secara publik di mana sebelumnya klien ini hanya tersedia bagi para pemilik [osu!supporter](https://osu.ppy.sh/home/support).
```

Comment (Nivalyx) [reply wiki/History_of_osu!/2014/id.md:31]: same as the above, this paragraph seems to have also been drastically altered/simplified in the english article as opposed to what it previously was

File: wiki/History_of_osu!/2014/id.md [hunk 7]
```diff
@@ -1,29 +1,44 @@
...
+Pada 1 Maret 2014, rancangan baru sistem *performance points* oleh Tom94 yang telah direncanakan pada bulan-bulan sebelumnya telah siap untuk ditetapkan untuk *gamemode* [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), dan [osu!mania](/wiki/Game_mode/osu!mania) sebagai sistem peringkat resmi untuk ketiga *gamemode* tersebut, dan juga peringatan bahwa sistem pada saat itu belum selesai secara 100%, dan masukan lebih lanjut dari berbagai pemain masih dibutuhkan \[4\].
+
+Saat elemen skinning bertambah banyak, elemen, `playfield.jpg` dihapus dan diperbarui dan sekarang tidak dapat diubah. `playfield.jpg` adalah elemen yang memungkinkan Anda untuk menggunakan gambar apa pun sebagai latar belakang bawaan *beatmap*, jika tidak ada yang disediakan oleh pembuat *map*. Fungsionalitas ini kemudian digunakan kembali dengan elemen `menu-background.jpg`, dimana elemen tersebut mengganti tampilan utama dan juga latar belakang default *beatmap* \[5,6\].
```
Comment (Nivalyx) [inline wiki/History_of_osu!/2014/id.md:11]:
```suggestion
Dari area *skinning*, elemen `playfield.jpg` yang sebelumnya dapat dimodifikasi resmi dipensiunkan dan diganti dengan gambar bawaan yang tidak dapat diubah. Elemen `playfield.jpg` merupakan elemen *skinning* yang memungkinkan para pemain untuk menggunakan gambar apa pun yang mereka inginkan sebagai latar belakang beatmap yang dimainkan apabila beatmap yang bersangkutan tidak memiliki latar belakang. Fungsionalitas ini kemudian diperkenalkan kembali melalui elemen baru `menu-background.jpg` yang mengubah tampilan gambar latar beatmap dan menu utama secara bersamaan \[5,6\].
```

File: wiki/History_of_osu!/2014/id.md [hunk 8]
```diff
@@ -1,29 +1,44 @@
----
-outdated_translation: true
----
+# Sejarah osu! 2014
 
-# 2014
+## Januari
+
+Seluruh sistem [*performance points* (pp)](/wiki/Performance_points) dinonaktifkan untuk proses pemeliharaan pada tanggal 20 Januari 2014 dikarenakan perhitungan yang salah yang terjadi pada beberapa bulan sebelumnya \[1\]. Penonaktifan tersebut diikuti oleh sebuah pengumuman pada tanggal 26 Januari 2014 yang menjelaskan rancangan ulang yang lengkap dan dimulai dari awal untuk sistem pp yang ditetapkan pada hari berikutnya \[2,3\]. Sistem baru tersebut dirancang dan diimplementasikan oleh [Tom94](https://osu.ppy.sh/users/1857058) dan bertujuan untuk menghitung skor pp dari sebuah "*difficulty value*" (nilai kesulitan) (terpisah dari [*star rating*](/wiki/Beatmapping/Star_rating)) yang ditentukan untuk semua kombinasi [*beatmap*](/wiki/Beatmap) dan [*mod*](/wiki/Game_modifier) dalam keseluruhan permainan osu!. Sebagai hasilnya, nama sistem pp yang lama diubah menjadi "ppv1" dan nama sistem pp yang baru dinamakan "ppv2".
```
[the remark this answers is not in the document]
Comment (rzkalqdr) [reply wiki/History_of_osu!/2014/id.md:5]: typo itu "berdasakan" wkwkw gue benerin ntar

File: wiki/History_of_osu!/2014/id.md [hunk 9]
```diff
@@ -1,29 +1,44 @@
...
+## April
 
-![](img/2014-03_01.jpg "playfield.jpg sudah tidak ada lagi")
+"osu! Monthly Fanart Contest" pertama kalinya diumumkan pada tanggal 3 April 2014. Kontes tersebut merupakan kontes informal yang diselenggarakan oleh [LuigiHann](https://osu.ppy.sh/users/1079) yang akan menjalankan kontes tersebut setiap bulan dan memberikan dua orang pemenang penghargaan berupa [osu!supporter](https://osu.ppy.sh/home/support) (satu pemenang dipilih oleh publik dan satu pemenang lagi dipilih oleh staf osu!) \[7\]. Hasil dari edisi pertama kontes disimpulkan dalam sebuah [*newspost*](https://osu.ppy.sh/home/news/2014-05-21-first-official-osu-fanart-contest-results) pada tanggal 21 Mei 2014, dimana *player* [kunny](https://osu.ppy.sh/users/3931585) dan [DumplingYumYum](https://osu.ppy.sh/users/1715930) masing-masing memenangkan juara satu dan dua.
 
-## Juli
+## Juni
 
-peppy menghapus batasan unduhan\*!
+Pada tanggal 18 Juni 2014, [peppy](https://osu.ppy.sh/users/2) mengumumkan bahwa batasan pengunduhan *beatmap* akan dihapus untuk semua *player*; namun, akan tetap ada "batasan kewarasan (sanity check)" yang akan mencegah *bot* dari pengunduhan beatmap dalam jumlah yang berlebihan \[9\].
```
[the remark this answers is not in the document]
Comment (rzkalqdr) [reply wiki/History_of_osu!/2014/id.md:21]: typo "menguduh" gue fix ntar

**Comment by rzkalqdr** [comment]
ready for a recheck I think @Nivalyx

**Review by Nivalyx — APPROVED** [review]
should be good to go from my side

File: wiki/History_of_osu!/2014/id.md [hunk 10]
```diff
@@ -1,29 +1,44 @@
...
+Pada 1 Maret 2014, sistem *performance point* untuk mode permainan [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), dan [osu!mania](/wiki/Game_mode/osu!mania) yang telah direncanakan berbulan-bulan sebelumnya oleh Tom94 resmi digunakan sebagai sistem peringkat resmi untuk ketiga mode permainan tersebut. Perubahan ini disertai dengan peringatan bahwa pengembangan sistem pp yang ada pada saat itu masih berjalan dan masukan lebih lanjut masih sangat dibutuhkan \[4\].
+
+Dari area *skinning*, elemen `playfield.jpg` yang sebelumnya dapat dimodifikasi resmi dipensiunkan dan diganti dengan gambar bawaan yang tidak dapat diubah. Elemen `playfield.jpg` merupakan elemen *skinning* yang memungkinkan para pemain untuk menggunakan gambar apa pun yang mereka inginkan sebagai latar belakang beatmap yang dimainkan apabila beatmap yang bersangkutan tidak memiliki latar belakang. Fungsionalitas ini kemudian diperkenalkan kembali melalui elemen baru `menu-background.jpg` yang mengubah tampilan gambar latar beatmap dan menu utama secara bersamaan \[5,6\].
+
+![](img/2014-03_01.jpg "Latar belakang bawaan untuk beatmap tanpa latar belakang yang spesifik*")
```
Comment (TPGPL) [inline wiki/History_of_osu!/2014/id.md:13]: is the stray `*` in tooltip intentional?

Comment (Nivalyx) [reply wiki/History_of_osu!/2014/id.md:13]: oh the `*` should definitely not be there yeah

**Review by TPGPL — APPROVED** [review]

**Pull request merged**